README.md

```diff
@@ -131,6 +131,7 @@ with:
   base-size: 200
   growth-rate: 2.0
   test-factor: 1.0
+  change-title: false
```

Comment:
Thanks for adding this, but please also add a section entitled `change-title` above. Search for `### test-factor` as an example.